Samples/DynamicDependencies/Basic/cpp-console-unpackaged/DynamicDependenciesBasic/DynamicDependenciesBasic.cpp

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright(c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// DynamicDependenciesBasic.cpp : This file contains the 'main' function. Program execution begins and ends there.
```

Comment:
not a very useful comment. replace with something that explains the point of the sample or delete.

Reply:
updated